pkg/utils/kubernetes/health/machinedeployment.go

```diff
@@ -51,6 +51,14 @@ func CheckMachineDeployment(deployment *machinev1alpha1.MachineDeployment) error
 			continue
 		}
 
+		isStrategyManualInPlace := deployment.Spec.Strategy.Type == machinev1alpha1.InPlaceUpdateMachineDeploymentStrategyType && deployment.Spec.Strategy.InPlaceUpdate != nil && deployment.Spec.Strategy.InPlaceUpdate.OrchestrationType == machinev1alpha1.OrchestrationTypeManual
```

Comment:
Can we not use `IsMachineDeploymentStrategyManualInPlace` here?

Reply:
It causes a cyclic dependency. Can you think of any other package where this helper function will be suitable to be in?

Reply:
Can't think of now. Can be done later.